search/searcher/search_geoboundingbox.go

```diff
@@ -22,6 +22,8 @@ import (
 	"github.com/blevesearch/bleve/search"
 )
 
+var GeoBitsShift = (geo.GeoBits << 1) - 1
```

Comment:
if the constant is just (geo.GeoBits << 1) then we could use this at one more place?
eg: inside relateAndRecurse() level computations.

Reply:
latest version has the GeoBits shift precomputed